v3/src/data-interactive/data-interactive-types.ts

```diff
@@ -31,6 +32,7 @@ export interface DIResources {
 }
 
 export interface DIValues {
+  attrs?: IAttributeArchive[]
```

Comment:
As mentioned elsewhere, we have a similar type `ICodapV2Attribute` in `codap-v2-types.ts`. If the DI usage is somewhat different perhaps that type could be extended/modified locally rather then defining an entirely new type.